src/ghost/ghost-post.service.ts

```diff
@@ -44,10 +45,25 @@ export class GhostPostService {
     }
 
     async updateArticleFromGhostPost(account: Account, ghostPost: GhostPost) {
-        const apId = account.getApIdForPost({
-            uuid: ghostPost.uuid,
-            type: PostType.Article,
-        });
+        const apIdForPost = await this.getApIdForGhostPost(ghostPost.uuid);
+        if (!apIdForPost) {
+            this.logger.info(
+                'No apId found for ghost post {uuid}, creating new post',
+                { uuid: ghostPost.uuid },
+            );
+            const newPostResult = await this.createGhostPost(
+                account,
+                ghostPost,
+            );
+            if (isError(newPostResult)) {
+                this.logger.error(
+                    'Failed to create new post with uuid: {uuid}, error: {error}',
+                    { uuid: ghostPost.uuid, error: getError(newPostResult) },
+                );
+            }
+            return;
```

Comment:
Are we ok to fail silently here?

Reply:
No. You are right, I have this on my list to clean up this `updateArticleFromGhostPost` method. This method doesn't return the errors back to the controller and we always return a 200 for updates. I'll clean this up in a separate PR.